src/shadow-ui/prepare-stylesheet.js

```diff
@@ -0,0 +1,27 @@
+import css from 'css';
+import SHADOW_UI_CLASS_NAME from './class-name';
+
+const ID_AND_CLASS_REG_EXP = /#[a-zA-Z0-9_-]+|\.-?[a-zA-Z0-9_][a-zA-Z0-9_-]*/g;
+const ADD_POSTFIX_PATTERN  = '$&' + SHADOW_UI_CLASS_NAME.postfix;
+
+
+function addUIClassPostfix (rules) {
+    rules.forEach(rule => {
+        if (rule.type === 'rule' && rule.selectors && rule.selectors.length)
+            rule.selectors = rule.selectors.map(selector => selector.replace(ID_AND_CLASS_REG_EXP, ADD_POSTFIX_PATTERN));
+
+        if (rule.rules)
+            addUIClassPostfix(rule.rules);
+    });
+}
+
+export default function prepareStylesheet (text) {
```

Comment:
I don't think this func name fits well. Maybe `convertToShadowStylesheet` (we need to change filename as well)? \cc @VasilyStrelyaev